sbi/inference/potentials/likelihood_based_potential.py

```diff
@@ -8,8 +8,8 @@
 from torch.distributions import Distribution
 
 from sbi.inference.potentials.base_potential import BasePotential
-from sbi.neural_nets.density_estimators import ConditionalDensityEstimator
-from sbi.neural_nets.density_estimators.shape_handling import (
+from sbi.neural_nets.estimators import ConditionalDensityEstimator
```

Comment:
I get the need to distinguish between density estimators, ratio estimators, and score estimators, but I am not sure about just "estimators" as the name. Also relates to #1190, which we should address before the release?